README.md

```diff
@@ -1,289 +1,15 @@
-# Dintero Checkout
```

Comment:
The README.md file should be for developers, while the README.txt file is for the wordpress store